src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketRepositoryClient.java

```diff
@@ -29,4 +34,16 @@
      * @return a client.
      */
     BitbucketWebhookClient getWebhookClient();
+
+    /**
+     * Gets all the open pull requests for the repository. The returned stream will make paged calls to Bitbucket to
+     * ensure that all pull requests are returned. Consumers are advised that this can return large amounts of data
+     * and are <strong>strongly</strong> encouraged to not collect to a list or similar before processing items, but
+     * rather process them as they come in.
+     *
+     * @param state the state of the pull requests to fetch, a value of null means fetch all pull requests
+     * @return a stream of all (potentially spanning multiple pages) open pull requests
```

Comment:
I think the documentation here is wrong too- it's a stream of all pull requests of the provided state right? (Open, merged, declined etc.)